### PR TITLE
redirect http://www.paddlepaddle.org/index.cn.html to home page, And …

### PR DESCRIPTION
**Pull request by jetfuel**
…switch language to Chinese.

### CODE REVIEW

File: portal/portal/urls.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
     # HOME PAGE URLS
     # ---------------
     url(r'^$', views.home_root, name='home'),
+    url(r'^index.cn.html$', views.cn_home_root, name='home'),
```
Comment (varunarora) [inline portal/portal/urls.py]: I think its safe to leave the trailing $ out, may be they had query params. Make this change only if you have to make another commit, else ignore

File: portal/portal/views.py [hunk 2]
```diff
@@ -306,6 +306,11 @@ def home_root(request):
         return render(request, 'index.html')
 
 
+def cn_home_root(request):
+    portal_helper.set_preferred_language(request, None, 'zh')
```
Comment (cs2be) [inline portal/portal/views.py]: redirect is the response, set response = redirect, and then pass to portal helper

File: portal/portal/urls.py [hunk 3]
```diff
@@ -28,6 +28,7 @@
     # HOME PAGE URLS
     # ---------------
     url(r'^$', views.home_root, name='home'),
+    url(r'^index.cn.html', views.cn_home_root, name='home'),
```
Comment (cs2be) [inline portal/portal/urls.py]: change the name to home_cn,  might be problematic is you use the same name
